alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/model/Alert.kt

```diff
@@ -121,9 +121,9 @@ data class Alert(
 
     companion object {
 
-        const val ALERT_ID_FIELD = "id"
+        const val ALERT_ID_FIELD = "alert_id"
```

Comment:
Whats the reason behind changing the name from `id` to `alert_id` ?

Reply:
We need to insert the same string into `alert_mapping.json`. In the file, there is `monitor_id` and `trigger_id`, so I wanted to follow the same format and ensure it is clear what the `id` is for.